packages/esm-patient-list-management-app/src/lists-table/lists-table.component.tsx

```diff
@@ -77,7 +75,6 @@ const ListsTable: React.FC<PatientListTableProps> = ({
   const [sortParams, setSortParams] = useState({ key: '', order: 'none' });
   const [searchTerm, setSearchTerm] = useState('');
   const responsiveSize = layout === 'tablet' ? 'lg' : 'sm';
-  const debouncedSearchTerm = useDebounce(searchTerm);
```

Comment:
I've also removed this debounce implementation. Because we're not hitting the backend when searching, we don't need to debounce the search term. We can just use the search term directly to filter the lists.